mono/mini/mini-s390x.c

```diff
@@ -4677,10 +4677,11 @@ mono_arch_output_basic_block (MonoCompile *cfg, MonoBasicBlock *bb)
 		case OP_MEMORY_BARRIER:
 			s390_mem (code);
 			break;
-#if USE_COOP_GC
 		case OP_GC_SAFE_POINT: {
```

Comment:
I have it defined but commented out. I can enable again if it's mature enough to try. 

-------- Original message --------
From: Ludovic Henry <notifications@github.com> 
Date: 2015/10/25  07:49  (GMT-05:00) 
To: mono/mono <mono@noreply.github.com> 
Subject: Re: [mono] [threads] Use runtime flag to enable cooperative suspend
  (#2169) 

In mono/mini/mini-s390x.c:
> @@ -4677,10 +4677,11 @@ mono_arch_output_basic_block (MonoCompile *cfg, MonoBasicBlock *bb)
>  		case OP_MEMORY_BARRIER:
>  			s390_mem (code);
>  			break;
> -#if USE_COOP_GC
>  		case OP_GC_SAFE_POINT: {

As a matter of fact, that should never be reached, as in https://github.com/mono/mono/blob/master/mono/mini/mini-ops.h#L1080, this opcode is not defined for s390x.

—
Reply to this email directly or view it on GitHub.